include/constants.py

```diff
@@ -381,21 +392,21 @@
 xbar_rd_pow_dyn = xbar_rd_pow
 xbar_wr_pow_dyn = xbar_wr_pow
 dac_pow_dyn = dac_pow_dyn_dict [str(cfg.dac_res)]
-adc_pow_dyn = adc_pow_dyn_dict [str(cfg.adc_res)]
+adc_pow_dyn = 0 #adc_pow_dyn_dict [str(cfg.adc_res)]
```

Comment:
Why make adc_pow_dyn 0 here, when the param.adc_pow_dyn is not used in hw_stats.py (hw_stats now is directly using entries from param.adc_pow_dyn_dict) ?

Reply:
Yes,  it will access the adc_pow_dyn_dict dictionary based on adc resolution. 
 param.adc_pow_dyn_dict[str(cfg.adc_res_new[str('matrix_adc_'+str(k))])]
Are you ok with that?